Libraries/Animated/src/NativeAnimatedHelper.js

```diff
@@ -166,5 +164,5 @@ module.exports = {
   generateNewNodeTag,
   generateNewAnimationId,
   assertNativeAnimatedModule,
-  supportsNativeListener,
```

Comment:
@janicduplessis just a guess, maybe FB is relying on this. We could stub it out as `return true` and try to merge.